Tests/KingfisherTests/KingfisherManagerTests.swift

```diff
@@ -85,10 +85,14 @@ class KingfisherManagerTests: XCTestCase {
                 XCTAssertNotNil(result.value?.image)
                 XCTAssertEqual(result.value!.cacheType, .none)
                 exp.fulfill()
-        }}}}}
+            }
+        }
+        }
+        }
+        }
```

Comment:
What about using Xcode auto indentation?
This way we can have the function `testRetrieveImage`  more organized

```swift
func testRetrieveImage() {
    let exp = expectation(description: #function)
    let url = testURLs[0]

    stub(url, data: testImageData)

    let manager = self.manager!

    manager.retrieveImage(with: url) { result in
        XCTAssertNotNil(result.value?.image)
        XCTAssertEqual(result.value!.cacheType, .none)

        manager.retrieveImage(with: url) { result in
            XCTAssertNotNil(result.value?.image)
            XCTAssertEqual(result.value!.cacheType, .memory)

            manager.cache.clearMemoryCache()

            manager.retrieveImage(with: url) { result in
                XCTAssertNotNil(result.value?.image)
                XCTAssertEqual(result.value!.cacheType, .disk)

                manager.cache.clearMemoryCache()

                manager.cache.clearDiskCache {
                    manager.retrieveImage(with: url) { result in
                        XCTAssertNotNil(result.value?.image)
                        XCTAssertEqual(result.value!.cacheType, .none)
                        exp.fulfill()
                    }
                }
            }
        }
    }

    waitForExpectations(timeout: 3, handler: nil)
}
```

Reply:
Agree, Xcode indentation would be better.